packages/utils/tests/conversions.test.ts

```diff
@@ -170,6 +174,40 @@ describe("Conversions", () => {
         })
     })
 
+    describe("# base64ToBuffer / bufferToBase64", () => {
+        const testBase64 = "SGVsbG8sIFdvcmxkIQ=="
```

Comment:
Also test some arbitrary byte data which isn't text?  You want this to be able to handle things like null bytes, control characters, etc.